src/components/Label/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Select from 'react-select'
+
+class SelectionLabel extends React.Component {
```

Comment:
You can directly put export here and remove the last line of export.
Just like this
`export class SelectionLabel extends React.Component {`